### DISCUSSION TITLE
Clicking "Done" doesn't dismiss preferences window

### DISCUSSION THREAD

**Discussion by geofftaylor** [ttscoff/bunch-docs#34] [Q&A] [answered]
In beta 74, clicking the "Done" button doesn't dismiss the preferences window (and it displays on top of all other windows, even if another app has focus). The only workaround I've found is quitting and restarting Bunch. (I'm using macOS 10.15.7 in case that makes a difference.)

**Comment by ttscoff** [comment, marked as answer]
Yep, just caught this this morning. B75 fixes and is out for automatic updates now. Sorry about that!

**Reply by ttscoff** [reply]
Ok, false start, b76 is out now and actually fixes it.
